excercsises/3_synchronization/pages/contact.js

```diff
@@ -8,13 +8,13 @@ var Contact = function () {
   this.getTitle = function () {
     return browser.driver.getTitle();
   };
-  this.fillFormSubmit  = (name, email, content,success) => {
+  this.fillFormSubmit  = (name, email, content) => {
     element(by.id("name")).clear().sendKeys(name);
     element(by.id("email")).clear().sendKeys(email);
     element(by.id("content")).clear().sendKeys(content);
     element(by.css(".test")).click();
   };
-  this.getSuccessInformation = ()=>{
+  this.findSuccessInformation = ()=>{
```

Comment:
I think 'getSuccessInformation' was a good name here. When your method is returning anything, text, object, array. Starting the name with 'get' is ok.